fixtures/vector/only_plaintext_profile_snapshot_version_100.json

```diff
@@ -46,6 +46,13 @@
         },
         "transaction": {
             "defaultDepositGuarantee": "0.975"
+        },
+        "resource": {
```

Comment:
good catch! actually I am gonna revert this change since this JSON isn't used to validate the `AssetsPreferences` and its corresponding test can deal without any `assets` entry on the JSON.